### PR TITLE
1809 presence of version meta

### LINKED ISSUE #1809: ParsingTrain.java:57-60: Add xsl transformation that...
The puzzle `1542-94d035b1` from #1542 has to be resolved: 

https://github.com/objectionary/eo/blob/8f4cff3e4b5fb18b05e03944c5c718bd9402cd44/eo-parser/src/main/java/org/eolang/parser/ParsingTrain.java#L57-L60

The puzzle was created by rultor on 16-Feb-23. 

Estimate: 90 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by levBagryansky**
resolves #1809

### CODE REVIEW

**Comment by levBagryansky** [comment]
@Graur please review

**Comment by levBagryansky** [comment]
@MikhailLipanin please review

File: eo-parser/src/main/java/org/eolang/parser/ParsingTrain.java [hunk 1]
```diff
@@ -89,6 +84,7 @@ public final class ParsingTrain extends TrEnvelope {
         "/org/eolang/parser/warnings/duplicate-metas.xsl",
         "/org/eolang/parser/warnings/mandatory-package-meta.xsl",
         "/org/eolang/parser/warnings/mandatory-home-meta.xsl",
+        "/org/eolang/parser/warnings/mandatory-version-meta.xsl",
```
Comment (MikhailLipanin) [inline eo-parser/src/main/java/org/eolang/parser/ParsingTrain.java:87]: We need to add `incorrect-version.xsl` to this pipeline too, like `incorrect-home` and `incorrect-architect.xsl`

**Review by MikhailLipanin — CHANGES_REQUESTED** [review]

**Review by MikhailLipanin — APPROVED** [review]
@levBagryansky Looks good to me,
@yegor256 can you merge?

**Comment by levBagryansky** [comment]
@yegor256 please take a look

**Review by Graur — APPROVED** [review]

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**